### ISSUE TITLE
Usage w/ Apollo Federation

### ISSUE THREAD

**Issue by dobrynin** [michalkvasnicak/aws-lambda-graphql#139]
Hello!

Thank you for developing this awesome tool. Have you considered at all how to make this solution work w/ Apollo Federation or microservices generally? Currently, Federation does not support subscriptions. Some folks have developed workarounds by using schema stitching to handle subscriptions. I haven't been able to find much discussion around combining the schema stitching approach w/ lambda based services.

Basically our ideal solution is to have a Fargate (long-running server, not lambda) server acting as the gateway to the rest of our GraphQL services, which are lambda based. Any idea how we could wire up aws-lambda-graphql to work thru an intermediary gateway?

**Comment by michalkvasnicak** [comment]
@dobrynin schema stitching as a workaround for subscriptions basically means that your gateway handles subscriptions and on each operation delegates this operation to microservice that handles it. So basically you will end up with the fact, that your gateway acts as a GraphQL client too. It isn't really automatic, you probably end up with programming whole delegation layer for your subscriptions.
